mondrian/src/it/java/mondrian/xmla/XmlaErrorTest.java

```diff
@@ -22,9 +22,9 @@
 import java.io.PrintStream;
 import java.util.*;
 import java.util.Enumeration;
-import javax.servlet.*;
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
+import jakarta.servlet.*;
```

Comment:
Import the specific classes instead of *. I know it was already there before but, just to follow the convention